### PR TITLE
Update cart

### PR DESCRIPTION
**Pull request by trongxt-0611**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52102)

## WHAT (optional)
- Remove item from cart
- Clear cart
- Update product quantity in cart

## HOW
- Use Jquery/Ajax

## Evidence (Screenshot or Video)
chrome-capture-2022-7-25
chrome-capture-2022-7-25 (1)

### CODE REVIEW

**Comment by haltn-3346** [comment]
oke

**Comment by trongxt-0611** [comment]
ready

File: config/routes.rb [hunk 1]
```diff
@@ -10,7 +10,10 @@
     end
     get "/menu", to: "home#menu"
     resources :products, controller: :home, only: :show
-    resource :carts
+    resource :carts, only: [:show, :create] do
+      get "/clear", to: "carts#clear"
+    end
+    resources :carts, only: [:update,:destroy]
```
Comment (ngocvt-0484) [inline config/routes.rb]:
```suggestion
    resources :carts, only: [:update, :destroy]
```

**Comment by trongxt-0611** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**